Test of hydroshare.org is failing

This was first observed when working in a [CI job](https://github.com/jupyterhub/repo2docker/actions/runs/29845120944/job/88683639248?pr=1504) when working on https://github.com/jupyterhub/repo2docker/pull/1504.

### Bug description

```
=========================== short test summary info ============================
FAILED tests/unit/contentproviders/test_hydroshare.py::test_content_id - AssertionError: assert 'b8f6eae9d892...1.v1783452415' == 'b8f6eae9d892...1.v1585005408'
  
  - b8f6eae9d89241cf8b5904033460af61.v1585005408
  ?                                     ^ ------
  + b8f6eae9d89241cf8b5904033460af61.v1783452415
  ?                                    ++++ ^^^
FAILED tests/unit/contentproviders/test_hydroshare.py::test_detect_hydroshare - AssertionError: assert {'resource': ... '1783452415'} == {'host': {'ho... '1585005408'}
  
  Omitting 2 identical items, use -vv to show
  Differing items:
  {'version': '1783452415'} != {'version': '1585005408'}
  
  Full diff:
    {
  +     'resource': 'b8f6eae9d89241cf8b5904033460af61',
        'host': {
            'hostname': [
                'https://www.hydroshare.org/resource/',
                'http://www.hydroshare.org/resource/',
                'https://hydroshare.org/resource/',
                'http://hydroshare.org/resource/',
            ],
            'django_s3': 'https://www.hydroshare.org/django_s3/download/bags/',
            'version': '[https://www.hydroshare.org/hsapi/resource/{}/scimeta/elements](https://www.hydroshare.org/hsapi/resource/%7B%7D/scimeta/elements)',
        },
  -     'resource': 'b8f6eae9d89241cf8b5904033460af61',
  -     'version': '1585005408',
  ?                   ^ ------
  +     'version': '1783452415',
  ?                  ++++ ^^^
    }
====== 2 failed, 255 passed, 2 xpassed, 12 warnings in 253.58s (0:04:13) =======
```

#### Expected behaviour

Test of hydroshare.org to pass.

#### Actual behaviour

Test of hydroshare.org fails.

### How to reproduce

```bash
pytest tests/unit/contentproviders/test_hydroshare.py
```

### Your personal set up

- OS: Linux
- Docker version: 29.5.3
- repo2docker version: 0.0.0+unknown

--- a/tests/unit/contentproviders/test_hydroshare.py
+++ b/tests/unit/contentproviders/test_hydroshare.py
@@ -37,7 +37,7 @@ def test_content_id():
     hydro = Hydroshare()
 
     hydro.detect("10.4211/hs.b8f6eae9d89241cf8b5904033460af61")
-    assert hydro.content_id == "b8f6eae9d89241cf8b5904033460af61.v1585005408"
+    assert hydro.content_id == "b8f6eae9d89241cf8b5904033460af61.v1783452415"
 
 
 def test_detect_hydroshare():
@@ -54,7 +54,7 @@ def test_detect_hydroshare():
             "version": "https://www.hydroshare.org/hsapi/resource/{}/scimeta/elements",
         },
         "resource": "b8f6eae9d89241cf8b5904033460af61",
-        "version": "1585005408",
+        "version": "1783452415",
     }
 
     assert (

